docs/DeveloperGuide.md

```diff
@@ -5,9 +5,261 @@
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 ## Design & implementation
```

Comment:
Is this an empty segment or meant to encapsulate the next section (Design)? If it is meant to encapsulate the next section, consider using different levels of heading. eg. ## for overarching segment, ### for sub segment